src/components/burgerConstructor/burgerConstructor.jsx

```diff
@@ -28,27 +29,45 @@ IngredientsCard.propTypes = {
   }),
 };
 
-function FinalPrice(props) {
+function FinalPrice() {
+  const data = useContext(Order);
+
   return (
     <p
       className={`${burgerConstructorStyles.price} text text_type_digits-medium`}
     >
-      {props.data.reduce((priv, elem) => priv + elem.price, 0)}
+      {data.reduce((priv, elem) => {
+        if (elem.type === "bun") return priv + elem.price * 2;
+        return priv + elem.price;
+      }, 0)}
```

Comment:
Супер, что для вычисления стоимости бургера используется метод reduce
Можно также воспользоваться еще и возможностями стрелочных функций)
```suggestion
      {data.reduce((priv, { type, price }) => type === "bun" ? priv + price * 2 : priv + price, 0)}
```